core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -39,6 +39,16 @@ class EmbeddedZookeeper() {
   def shutdown() {
     CoreUtils.swallow(zookeeper.shutdown())
     CoreUtils.swallow(factory.shutdown())
+
+    def isDown(): Boolean = {
```

Comment:
There are a few tests that only extend ZooKeeperTestHarness. Do we need to ensure that ZK is down when shutting down ZooKeeperTestHarness?